llvm/lib/Transforms/Instrumentation/IndirectCallPromotion.cpp

```diff
@@ -851,8 +859,27 @@ bool IndirectCallPromoter::isProfitableToCompareVTables(
     LLVM_DEBUG(dbgs() << "\n");
 
     uint64_t CandidateVTableCount = 0;
-    for (auto &[GUID, Count] : VTableGUIDAndCounts)
+    SmallVector<MDNode *, 2> Types;
```

Comment:
We can move this to line L870 and drop the `clear()`. Since it's a small vector with only 2 pointers (on the stack) I don't expect changing this will affect performance.